### PR TITLE
Added automatically derived docs

### PR DESCRIPTION
**Pull request by Buckram123**
`#![warn(missing_docs)]` shouldn't warn inside derived traits/impls

### CODE REVIEW

**Review by CyberHoward — APPROVED** [review]
Awesome!

**Review by Kayanski — APPROVED** [review]

File: packages/cw-orch-fns-derive/src/execute_fns.rs [hunk 1]
```diff
@@ -148,7 +148,6 @@ pub fn execute_fns_derive(input: DeriveInput) -> TokenStream {
 
     let derived_trait_impl = quote!(
         #[automatically_derived]
-        #[allow(undocumented)]
```
Comment (CyberHoward) [inline packages/cw-orch-fns-derive/src/execute_fns.rs]: Why did you remove this?

Comment (Buckram123) [reply packages/cw-orch-fns-derive/src/execute_fns.rs]: `automatically_derived` covers it for impl blocks!

Comment (Buckram123) [reply packages/cw-orch-fns-derive/src/execute_fns.rs]: Actually it won't warn on trait impls, since docs of it should be inside the trait itself
